.gitignore

```diff
@@ -0,0 +1,10 @@
+# ignore  all .a files
+*.md
+
+# but do track README.md, even though you're ignoring .md files above
+!README.md
```

Comment:
- [ ] I would suggest removing the `README.md` file and the `*.md` from the `.gitignore`, since we actually want the README and the .md files in the repository :smiley:
- [ ] One of the project requirements is to ignore the `test.md` file. Please add `test.md` in your .gitignore file.

![image](https://user-images.githubusercontent.com/81522435/138046729-608c0617-7fae-4440-adbc-9749f627301c.png)